examples/fwdAttr/FwdServer.py

```diff
@@ -0,0 +1,56 @@
+import numpy
+import logging
+from numpy import void
+
+import gevent
+
+import tango
+from tango.server import run
+from tango.server import Device
+from tango.server import attribute, command
+from tango.server import class_property, device_property
+from tango import AttrQuality, AttrWriteType, DispLevel, DevState
+
+__all__ = ["FwdServer", "main"]
+
+class FwdServer(Device):
+    """
+    Start this server: python FwdServer.py myFwdServer
+    Start the server containing the root attribute.
+    Then using jive select the Attribute properties from myFwdServer
+    Select you forwarded attribute and add the value to __root_att
+    e.g. __root_att   ->  x/y/z/root_attribute_name
+    Now restart the FwdServer
+    """
+
+    def __init__(self, *args, **kwargs):
+        Device.__init__(self, *args, **kwargs)
```

Comment:
I think that this is unnecessary, unless you specifically want to follow a policy that every class must have a `__init__` method implemented.